project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.34)
+message(FATAL_ERROR "VAAPI minimum version 0.34 not found")
+endif()
+
 if(NOT VAAPI_FIND_VERSION)
   set(VAAPI_FIND_VERSION 0.38.0)
```

Comment:
From my point of view, the 4 lines you added here are not necessary and should go out since we're already requiring 0.38.0. If 0.34.0 is the minium version you should set that in line 33.
But let @wsnipex confirm that, I'm not sure I understand the problem he sees.